gfop/get_food_counts.py

```diff
@@ -121,7 +129,11 @@ def get_dataset_food_counts(gnps_network, metadata, filename_col,
     """
     food_counts, filenames = [], []
     gnps_network = pd.read_csv(gnps_network, sep='\t')
-    sample_types = get_sample_types(sample_types)
+    if ref_metadata != 'gfop':
+        delim = ',' if ref_metadata.endswith('.csv') else '\t'
```

Comment:
I think a better interface is to work with Pandas dataframes rather than filenames. There is such a wide variety in how data can be stored in a text file that it's impossible to cover all cases that are going to happen with such a simple `if` statement.

This is a bigger design question than should be addressed in this PR, but I wanted to mention it so we can start thinking about what the most user-friendly and flexible approach is.

Reply:
Ok noted